src/Articulate.Web/Web.config

```diff
@@ -61,14 +61,14 @@
   </appSettings>
   <connectionStrings>
     <remove name="umbracoDbDSN" />
-  </connectionStrings>
+  <add name="umbracoDbDSN" connectionString="Data Source=|DataDirectory|\Umbraco.sdf;Flush Interval=1;" providerName="System.Data.SqlServerCe.4.0" /></connectionStrings>
```

Comment:
Changes to those files should have not been pushed. Database connection string or a fresh clone will not trigger the installation of umbraco